src/simulators/qasm/qasm_controller.hpp

```diff
@@ -186,13 +200,22 @@ class QasmController : public Base::Controller {
   // Initial statevector for Statevector simulation method
   cvector_t initial_statevector_;
 
+  std::vector<std::shared_ptr<CircuitOptimization>> optimizations;
```

Comment:
`optimizations_;` with underscore for private/protected class member

Also could you add a short documentation comment above for what this is.